README.md

```diff
@@ -122,6 +122,7 @@ First of all, `headers` in axios are request headers, not response headers. Ther
 - CORS problems are browser only, when your site requests a resource that has a different origin (domain, protocol, or port) from its own. Node.js scripts and [Postman][postman] don't have this kind of trouble.
 - Sometimes, take it easy for those additional OPTIONS requests. They are [preflighted requests](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#Preflighted_requests), which are very normal things and not strange bugs caused by axios.
 - If some headers couldn't be accessed in your codes, even though they were visible in the network panel, please make sure the server responses correct [Access-Control-Expose-Headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#Access-Control-Expose-Headers) header.
+- If the above method is not working then you have to set the value of `access-control-expose-headers` to the header that you want to expose to the axios in Response Headers issued from the server.
```

Comment:
Wow, the first contributor comes. 🥳 

I think line 124 has explained the same thing, right?

Reply:
@chinesedfan  aah.. you are right.  It's a blunder from me. Sorry

Reply:
Haha, never mind. At least we thought together, including the inserted position.